src/tutorials/getting-started-with-drizzle-and-react.md

```diff
@@ -231,8 +231,6 @@ This should create a `client` directory in your Truffle project and bootstrap a
 
```

Comment:
I propose taking out the heading "Setting up the front-end client" and replacing it with something like "Wiring up the front-end client" or "Linking up our build artifacts", then remove the following:

`Now that we have a front-end client located inside the `client` directory, change into that directory with the command `cd client` and continue with the following steps to set it up.`

That way the next directions are to go into `truffle-config.js` and link the build dir to `client/src/contracts`